src/python/tests/core/compare_core_model.py

```diff
@@ -0,0 +1,75 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Check the current ensembl-py core model against a core created from Ensembl SQL.
+This script gets one row for each table in the ORM to check that SQLalchemy can correctly query the table.
+If not, it will show the OperationalError exception to explain what is wrong in the ORM.
+
+Use this script this check the ORM and fix it.
```

Comment:
```suggestion
This script gets one row for each table in the ORM to check that SQLAlchemy can correctly query the table.
If not, it will show the `OperationalError` exception to explain what is wrong in the ORM.

Use this script to check the ORM (and fix it if needed).
```